frontend/apps/app/components/Chat/services/aiMessageService.ts

```diff
@@ -33,6 +33,9 @@ interface SendChatMessageResult {
  */
 const ChatAPIResponseSchema = object({
   success: optional(boolean()),
+  jobId: optional(string()),
+  status: optional(string()),
+  message: optional(string()),
```

Comment:
These values are not used on the front end, correct? If so, I would like to eliminate them. I would like the PR to be issued with as little code as possible.

Reply:
As you said, it was an unnecessary response.
I deleted it.
https://github.com/liam-hq/liam/pull/1993/commits/77b6c8ebc2c0991a65c33b91c1d9635b82fbeb77